docs/pages/guides/migrating-to-react-native-firebase.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Migrating from Expo Firebase to React Native Firebase
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+The `expo-firebase-analytics` and `expo-firebase-recaptcha` packages allowed the use of some Firebase features on Classic Builds, Expo Go, and Expo for Web, but developers often encountered version conflicts when trying to use them alongside the rest of the Firebase suite. With EAS Build and Development Builds, it is much easier to use [React Native Firebase](https://rnfirebase.io/) directly in your entire development workflow, so these packages have been deprecated.
+
+In this guide, we will migrate from the Expo Firebase packages to React Native Firebase by:
+- Replacing the Expo Firebase packages with their React Native Firebase equivalents
+- - Replacing the use of Expo Go for testing with [Development Builds](/development/introduction/) customized to your project's needs
+- Testing the updated code on Development Builds
```

Comment:
```suggestion
- Replacing the Expo Firebase packages with their React Native Firebase equivalents
- Replacing the use of Expo Go for testing with [development build](/development/introduction/) customized to your project's needs
- Testing the updated code on development build
```

Reply:
Removing the extra `-` and updating instances of "Development Builds" to "development build" for consistency reasons. Earlier, when I reviewed the draft, I wasn't aware of this. Sorry, for the inconvenience caused here 🙏